### PR TITLE
Feat/wally alpha

### PR DESCRIPTION
**Pull request by dnikitop**

### CODE REVIEW

File: base_local_planner/src/map_grid.cpp [hunk 1]
```diff
@@ -339,5 +339,19 @@ namespace base_local_planner{
       }
     }
   }
+  bool MapGrid::worldToMap(double wx, double wy, unsigned int& mx, unsigned int& my, double resolution) const
+  {
+    if (wx < origin_x_ || wy < origin_y_)
```
Comment (grieneis) [inline base_local_planner/src/map_grid.cpp:346]: use {} for all if statements

File: base_local_planner/src/map_grid.cpp [hunk 2]
```diff
@@ -339,5 +339,19 @@ namespace base_local_planner{
...
+      return false;
+
+    mx = (int)((wx - origin_x_) / resolution);
+    my = (int)((wy - origin_y_) / resolution);
+
+
+    if (mx < size_x_ && my < size_y_)
+      return true;
```
Comment (grieneis) [inline base_local_planner/src/map_grid.cpp:357]: use {} for all if statements

**Review by grieneis — DISMISSED** [review]
One comment

File: base_local_planner/src/map_grid.cpp (hunk 2 above)
Comment (27Apoorva) [reply base_local_planner/src/map_grid.cpp:357]: done

**Review by 27Apoorva — APPROVED** [review]

**Pull request merged**